src/lib/controllers/ui-controller.js

```diff
@@ -1003,7 +1003,7 @@ export default class UIController {
     return this
   }
 
-  newLexicalRequest (targetWord, languageID) {
+  newLexicalRequest (targetWord, languageID, data=null) {
```

Comment:
Maybe it's better to provide a more descriptive name to indicate what type of data it is? Is it a text selector data?

Reply:
good idea, yes its text selector data. Will fix.